meson.build

```diff
@@ -116,6 +116,29 @@ if get_option('enable-tensorflow-lite')
   endif
 endif
 
+# Python
+prog_pkgconfig = find_program('pkg-config')
+have_python = false
+have_python3 = false
+
+if get_option('enable-python')
+  python_dep = dependency('python-2.7', required: false)
```

Comment:
python2 looks more general than python-2.7.

Reply:
Python 2 is equivalent to Python 2.7. So, we support python 2.7 & 3.x.